src/commands/initProjectForVSCode/InitVSCodeLanguageStep.ts

```diff
@@ -22,21 +22,24 @@ export class InitVSCodeLanguageStep extends AzureWizardPromptStep<IProjectWizard
 
     public async prompt(context: IProjectWizardContext): Promise<void> {
         // Display all languages, even if we don't have full support for them
-        const languagePicks: QuickPickItem[] = [
-            { label: ProjectLanguage.CSharp },
-            { label: ProjectLanguage.CSharpScript },
-            { label: ProjectLanguage.FSharp },
-            { label: ProjectLanguage.FSharpScript },
-            { label: ProjectLanguage.Java },
-            { label: ProjectLanguage.JavaScript },
-            { label: ProjectLanguage.PowerShell },
-            { label: ProjectLanguage.Python },
-            { label: ProjectLanguage.TypeScript },
-            { label: ProjectLanguage.Custom }
+        const languagePicks: IAzureQuickPickItem<{ language: ProjectLanguage, model?: number }>[] = [
+            { label: ProjectLanguage.CSharp, data: { language: ProjectLanguage.CSharp } },
```

Comment:
These values (or at least, similar) are used in multiple places; we should make a constant for them. That might eliminate the need for the type definition just above if it only remains in one place...

Reply:
Agree there seems to be some repetition/overlap in areas of the codebase, though there are some subtle differences.  For this PR, still being new to the codebase, I was trying not to do too much refactoring in hopes of not introducing major regressions.  There is a similar new model for Node.js coming down the road that will help to show where some of that commonality lies and be a good opportunity for further refactoring.